schemas/expconf/v0/log-policy.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://determined.ai/schemas/expconf/v0/log-policy.json",
+    "title": "LogPolicy",
+    "additionalProperties": false,
```

Comment:
I don't really know what best practices look like for future-proofing expconfs. Is it best to leave properties open or closed? Is it conditional? This PR has a mix.

Reply:
```additionalProperties``` I think should always be false for objects.

Where does this PR have a mix? ```additionalProperties``` is not supported for arrays which is why ```log-pattern-policies.json``` doesn't have it.

Reply:
Oh! That makes total sense. _of course_ it only makes sense of objects.

Thanks!